test/src/main/scala/scalarules/test/junit/test_discovery/ArchiveEntriesTest.scala

```diff
@@ -0,0 +1,25 @@
+package io.bazel.rulesscala.test_discovery
+
+import io.bazel.rulesscala.test_discovery.ArchiveEntries.listClassFiles
+import org.specs2.mutable.SpecWithJUnit
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+
+class ArchiveEntriesTest extends SpecWithJUnit {
+  "List only class files from a directory" in {
+    val dir = Files.createTempDirectory("temp")
+    Files.createFile(Paths.get(dir.toString, "SomeFile"))
```

Comment:
I think instead of `Paths.get(dir.toString, "SomeFile")` you could use `dir.resolve("SomeFile")`